keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -344,9 +354,11 @@ class ApplicationService(
                   status
                 )
                   ?.also {
+                    //todo eb: use only the last one
```

Comment:
I had to read this a couple of times to understand what it meant. Maybe something like "consolidate constraint summaries into a single list"?